lib/mocktopus.rb

```diff
@@ -1,5 +1,9 @@
 require 'logger'
-LOGGER = Logger.new('mocktopus.log', 'daily')
+
+$logger = Logger.new(STDOUT) unless $logger
+
+class ::Logger; alias_method :write, :info; end
```

Comment:
Yessir.  And I like the term monkeyhacks.  We need to redirect the use of logger puts and write methods to formatted logger messages (this will be handy for logstash/grok.  I think Sinatra is using write for errors, so I redirect those to logger.error.  

Sent from my iPhone

> On Mar 28, 2015, at 07:46, Mike Creque <notifications@github.com> wrote:
> 
> In lib/mocktopus.rb:
> 
> > @@ -1,5 +1,9 @@
> >  require 'logger'
> > -LOGGER = Logger.new('mocktopus.log', 'daily')
> > +
> > +$logger = Logger.new(STDOUT) unless $logger
> > +
> > +class ::Logger; alias_method :write, :info; end
> are these aliases still needed since logger is sending everything to stdout now? I'm assuming this is part of the mentioned monkeyhacks!
> 
> —
> Reply to this email directly or view it on GitHub.
>